### ISSUE TITLE
3 Fonts are downloaded instead of one ?

### ISSUE THREAD

**Issue by kevinmarrec** [Developmint/nuxt-webfontloader#4]
Using this configuration in `nuxt.config.js` :
```js
webfontloader: {
  google: {
    families: ['Roboto:300,400,500,700']
  }
}
```

I tried to build a simple app for production and run it with `nuxt build` then `nuxt start`.

And It seems that with the `nuxt-webfontloader`, the font is downloaded 3 times with a different name 😄
![screenshot_1](https://user-images.githubusercontent.com/25272043/49024828-ae84da80-f19a-11e8-90ba-4b0b1774e3f0.png)

Using the classic way without the module (using Nuxt config `head` object) results in :
![screenshot_2](https://user-images.githubusercontent.com/25272043/49025497-1d166800-f19c-11e8-9218-8af14694a851.png)

**Comment by kevinmarrec** [comment]
@manniL Yeah as this module is just a wrapper for Nuxt, it must be an issue of the official webfontloader. I'm not using `nuxt-pwa`.

Here is a better view of my Network tab with the Waterfall :
![screenshot_3](https://user-images.githubusercontent.com/25272043/49028442-b6e11380-f1a2-11e8-877b-855bd1cff669.png)

It seems that the first downloaded font is the good one and retrieved just after loading the css file form Google, the 2 others are called/downloaded from a `.js` file.

**Comment by manniL** [comment]
@kevinmarrec Did you use modern mode?

**Comment by manniL** [comment]
@kevinmarrec NVM.

Look at the stylesheet (example for Lato):

```css
/* latin-ext */
@font-face {
  font-family: 'Lato';
  font-style: normal;
  font-weight: 400;
  src: local('Lato Regular'), local('Lato-Regular'), url(https://fonts.gstatic.com/s/lato/v14/S6uyw4BMUTPHjxAwXjeu.woff2) format('woff2');
  unicode-range: U+0100-024F, U+0259, U+1E00-1EFF, U+2020, U+20A0-20AB, U+20AD-20CF, U+2113, U+2C60-2C7F, U+A720-A7FF;
}
/* latin */
@font-face {
  font-family: 'Lato';
  font-style: normal;
  font-weight: 400;
  src: local('Lato Regular'), local('Lato-Regular'), url(https://fonts.gstatic.com/s/lato/v14/S6uyw4BMUTPHjx4wXg.woff2) format('woff2');
  unicode-range: U+0000-00FF, U+0131, U+0152-0153, U+02BB-02BC, U+02C6, U+02DA, U+02DC, U+2000-206F, U+2074, U+20AC, U+2122, U+2191, U+2193, U+2212, U+2215, U+FEFF, U+FFFD;
}
/* latin-ext */
@font-face {
  font-family: 'Lato';
  font-style: normal;
  font-weight: 700;
  src: local('Lato Bold'), local('Lato-Bold'), url(https://fonts.gstatic.com/s/lato/v14/S6u9w4BMUTPHh6UVSwaPGR_p.woff2) format('woff2');
  unicode-range: U+0100-024F, U+0259, U+1E00-1EFF, U+2020, U+20A0-20AB, U+20AD-20CF, U+2113, U+2C60-2C7F, U+A720-A7FF;
}
/* latin */
@font-face {
  font-family: 'Lato';
  font-style: normal;
  font-weight: 700;
  src: local('Lato Bold'), local('Lato-Bold'), url(https://fonts.gstatic.com/s/lato/v14/S6u9w4BMUTPHh6UVSwiPGQ.woff2) format('woff2');
  unicode-range: U+0000-00FF, U+0131, U+0152-0153, U+02BB-02BC, U+02C6, U+02DA, U+02DC, U+2000-206F, U+2074, U+20AC, U+2122, U+2191, U+2193, U+2212, U+2215, U+FEFF, U+FFFD;
}
```

This loads fonts for latin and latin extend.

Quote from the npm package.

>If you need to specify character subsets other than the default (e.g.: greek script in addition to latin), you must append the subset string to the requested family string after a colon. The subset string should follow the Google documentation (subset names separated by commas):
```js
WebFontConfig = {
  google: {
    families: ['Open Sans Condensed:300,700:latin,greek']
  }
};
```

**Comment by manniL** [comment]
**tl;dr**

```js
webfontloader: {
    google: {
      families: ['Lato:400,700:latin'] //set the subset here
    }
  },
```
